src/mscorlib/shared/System/Runtime/CompilerServices/RuntimeWrappedException.cs

```diff
@@ -25,7 +28,19 @@ public RuntimeWrappedException(Object thrownObject)
 
         public override void GetObjectData(SerializationInfo info, StreamingContext context)
         {
+            if (info == null)
+            {
+                throw new ArgumentNullException(nameof(info));
+            }
+            Contract.EndContractBlock();
```

Comment:
@danmosemsft just went through and deleted all of these... probably shouldn't add more back :)

Reply:
Thanks, that's a good catch. Also the ArgumentNullException throws can be removed as the base impl of the method which needs to get called first handles that.